### DISCUSSION TITLE
Give OpenGothic an icon/logo

### DISCUSSION THREAD

**Discussion by Newbytee** [Try/OpenGothic#403] [Ideas]
If OpenGothic is to be installable as a GUI application, it would be good to have an icon/logo to use for shortcuts. Unfortunately, I'm not good at graphic design, so I can't really propose anything.

**Comment by Try** [comment]
Thank for ticket @Newbytee !

I'm also, was thinking same way for quite some time. There is one very icon on internet: https://www.deviantart.com/danilorosari/art/Gothic-Icon-Pack-628387657, but I don't know how to reach the author and ask him for *.ico and usage permission.

**Comment by Newbytee** [comment]
DeviantArt does have a feature where you can chat with people, but I'm not sure I see any activity on his profile since 2016, so it may be hard to reach him.

That said, the icon you linked does have a licence: https://creativecommons.org/licenses/by-nc-nd/3.0. It's not as liberal as the MIT licence, but assets don't necessarily have to be the same licence as the code for it to be considered "free software" by the Free Software Foundation if you care about that. The most noteworthy parts of the licence are that it doesn't permit commercial use and you're not allowed to make derivative works of it and then redistribute those.

**Comment by PolyMeilex** [comment]
Here are a couple of my drafts that I quickly put together. I'm not really happy with any of them, but In case you are interested I'll share them here. 

EDIT: Obviously if you are interested, I'll polish things up and share the design files. 

|   |   |   |
|---|---|---|
| ![img](https://i.imgur.com/e1wC57Q.png)  | ![img](https://i.imgur.com/xrOkyrh.png)  |  ![img](https://i.imgur.com/LmgJtLW.png)  

|   |   |   |
|---|---|---|
|![img](https://i.imgur.com/HsaK87Z.png) | ![img](https://i.imgur.com/nXp5gdw.png) | ![img](https://i.imgur.com/4szKJUG.png)

|   |   |   |
|---|---|---|
| ![img](https://i.imgur.com/LbD4m3P.png)| ![img](https://i.imgur.com/ZUdyE3X.png) | ![img](https://i.imgur.com/ATkA17y.png)

**Comment by clort81** [comment]
Websearch finds some existing icons people made.
[TheDoctor45](https://www.deviantart.com/thedoctor45/art/Gothic-Custom-Icon-211304556)
[Tarvanis](https://www.deviantart.com/tarvanis/art/Gothic-II-Desktop-Icon-772165343)
Just 'G' or maybe 'OG'?

**Comment by NefariusGM** [comment]
I suggest to use both "O" and "G" in simple, but creative way. As an icon, it should also scale well.

Something, like this: (my own work)

![OpenGothic](https://user-images.githubusercontent.com/104260066/164894381-19441d2e-7c39-44e3-ab76-93eb0d88a6ce.png)
